src/audio/Audio.js

```diff
@@ -158,7 +158,7 @@ class Audio extends Object3D {
 
 	}
 
-	stop() {
+	stop( delay = 0 ) {
```

Comment:
Do you mind updating the documentation with the new parameter?

https://threejs.org/docs/index.html#api/en/audio/Audio.stop

Maybe: 

> delay (optional) - The delay, in seconds, at which the sound should stop playing. <br />

Reply:
Sure

Reply:
Not sure what is the flow for other languages
`en` docs update https://github.com/mrdoob/three.js/pull/29376/commits/b8e90aa841abdc7ca5ef6ccfa8ed637fea079e09

Reply:
Let's ignore them for now and focus on `en`.